seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/protocol/codec/SeaTunnelSubmitJobCodec.java

```diff
@@ -37,30 +40,40 @@
  * to seatunnel-engine/seatunnel-engine-core/src/main/resources/client-protocol-definition/SeaTunnelEngine.yaml
  */
 
-@Generated("ebea440b36898863958c102f47603fee")
+/** */
+@Generated("9933654790f5fbe98d0ee1c248bc999b")
 public final class SeaTunnelSubmitJobCodec {
     // hex: 0xDE0200
     public static final int REQUEST_MESSAGE_TYPE = 14549504;
     // hex: 0xDE0201
     public static final int RESPONSE_MESSAGE_TYPE = 14549505;
     private static final int REQUEST_JOB_ID_FIELD_OFFSET =
             PARTITION_ID_FIELD_OFFSET + INT_SIZE_IN_BYTES;
-    private static final int REQUEST_INITIAL_FRAME_SIZE =
+    private static final int REQUEST_IS_START_WITH_SAVE_POINT_FIELD_OFFSET =
             REQUEST_JOB_ID_FIELD_OFFSET + LONG_SIZE_IN_BYTES;
+    private static final int REQUEST_INITIAL_FRAME_SIZE =
+            REQUEST_IS_START_WITH_SAVE_POINT_FIELD_OFFSET + BOOLEAN_SIZE_IN_BYTES;
     private static final int RESPONSE_INITIAL_FRAME_SIZE =
             RESPONSE_BACKUP_ACKS_FIELD_OFFSET + BYTE_SIZE_IN_BYTES;
 
     private SeaTunnelSubmitJobCodec() {}
 
     public static class RequestParameters {
 
+        /** */
```

Comment:
What does this do?

Reply:
It generated by hazelcast script. Removed.